assets/scss/modules/_tooltip.scss

```diff
@@ -116,3 +116,82 @@
     }
   }
 }
+
+/* New Tooltip II */
+
+@mixin tooltipDialog {
+  .tooltip {
+    display: block;
+    position: relative;
+    top: -50px;
+    left: 50%;
+    transform: translate(-50%, -50%) scale(0.75) rotate(15deg);
+    transform-origin: bottom center;
+    padding: $dp-spaces--lv3 $dp-spaces--lv4;
+    border-radius: 5px;
+    background: rgba(0, 0, 0, 0.75);
+    text-align: center;
+    transition: 0.15s ease-in-out;
+    opacity: 0;
+    width: 100%;
+    pointer-events: none;
+    z-index: 5;
+    text-align: left;
+    border: 1px solid $dp-color-silver;
+    box-shadow: 0 0 0 4px rgba(204, 204, 204, 0.3);
+
+    &.bgColor {
+      background: rgba($dp-color-white, 0.95);
+    }
+
+    &:before {
+      content: '';
+      display: block;
+      margin: $dp-spaces--lv0 auto;
+      width: $dp-spaces--lv0;
+      height: $dp-spaces--lv0;
+      border: 5px solid transparent;
+      position: absolute;
+      bottom: $dp-spaces--lv0;
+      left: 50%;
+      transform: translate(-50%, 100%);
+      border-top: 5px solid rgba($dp-color-white, 0.95);
+    }
+
+    p {
```

Comment:
it's ok the styles of the content inside the tooltip are here? in that case always `p, span` has this styles

Reply:
For me they are fine, because these elements have particular characteristics of this component